app/assets/locales/android_translatable_strings.txt

```diff
@@ -285,6 +285,7 @@ mult.install.bad=The selected ZIP file is not valid. Please choose a valid zip f
 mult.install.progress.baddest=Couldn't write multimedia to the local filesystem at: ${0}
 mult.install.progress.badentry=There was a bad entry in the zip file: ${0}
 mult.install.progress.errormoving=There was a problem copying the multimedia from the zip file, please try again.
+mult.install.progress.invalid.entry=The path of the entry ${0} doesn't match the parent folder, review the content of the ZIP file and try again!

```

Comment:
Probably just say `Invalid CCZ selected`. User would not necessary understand `entry` or `zip` here. So lets obscure the details because this issue would probably only surface when someone is deliberately trying to do something malicious.

Reply:
@avazirna this is still pending on this.

Reply:
I don't know what you mean @shubham1g5 😅. That's on me, I was waiting for the jobs to complete to push the new changes and then I forgot